lib/colorado_lottery.rb

```diff
@@ -0,0 +1,32 @@
+require "./lib/contestant"
+require "./lib/game"
+require "pry"
+
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+  end
+
+  def current_contestants
+    @registered_contestants
+  end
+
+  def interested_and_18?(contestant, game)
+    contestant.game_interests.include?(game.name) && contestant.age >= 18
+  end
+
+  def can_register?(contestant, game)
+    interested_and_18?(contestant, game) && (game.national_drawing? || contestant.out_of_state? == false)
+  end
+
+  def register_contestant(contestant, game)
+    if can_register?(contestant, game)
```

Comment:
This is a good start, but we need `contestant` to be in an array, and we need a way to add to that array if the `game.name` key already exist. 
```
def register_contestant(contestant, game)
    if registered_contestants[game.name]
      registered_contestants[game.name] << contestant
    else
      registered_contestants[game.name] = [contestant]
    end
  end
```
The above is one way to accomplish that.